pkg/logutil/redact.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package logutil
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var _enableRedactLog bool
```

Comment:
```suggestion
var enableRedactLog bool
```

Why need a `_` prefix?